src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -1358,6 +1379,25 @@ public static unsafe SocketError GetSockOpt(SafeSocketHandle handle, SocketOptio
             return GetSocketErrorForErrorCode(err);
         }
 
+        public static unsafe SocketError GetRawSockOpt(SafeSocketHandle handle, int optionLevel, int optionName, Span<byte> optionValue, ref int optionLength)
+        {
+            Debug.Assert((uint)optionLength <= optionValue.Length);
+
+            int optLen = optionLength;
+            fixed (byte* pinnedValue = &optionValue[0])
```

Comment:
Wouldn't this throw `IndexOutOfRangeException` if somebody passes a zero-length span? (I don't know why anybody would ever do this other than if they're using the existence of a raw option as a flag with no associated value.)

Reply:
Yup, muscle memory, didn't mean to add the [0].